README.md

```diff
@@ -79,6 +80,11 @@ Have you:
 	- Emails should have a meaningful subject.
 - [ ] Handed in the assignment before the due time (Friday tutorial 9am)?
 
+#### Active Participation (3%)
+
+A small component of the course assessment will be for active participation in practicals and tutorials.
+Since the majority of the course material is open source, issues filed against found in practicals, tutorials and lectures, and contributions to fixing those errors will count to this component.
```

Comment:
See, I wasn't even trying: "issues filed against found in practicals". Missing word.